src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -1212,6 +1210,14 @@ uct_ib_mlx5_devx_device_mem_free(uct_md_h uct_md, uct_mem_h memh)
         status = UCS_ERR_BUSY;
     }
 
+    uct_ib_mlx5_devx_obj_destroy(ib_memh->dm_addr_dvmr, "DM Address, KSM");
```

Comment:
1. "DM Address, KSM" -> "DM-KSM"
2. Destroy order should be the reverse of create order - first, destroy devx object